docs(platform-sdk): coin spec updates

sleepdefic1t opened the pull request:
## Summary

- Platform-SDK Updates
  - add AVAX Spec
  - add DOT Spec
  - update ADA Spec
  - update ARK Spec
  - remove XMR Spec files

## Checklist

- [x] Ready to be merged

mvega3 commented:
@JeremiGendron, these changes are good and very much welcomed -- just update your branch to reflect the latest changes to `master` and your PR will be accepted.

JeremiGendron commented:
Cool, will do @mvega3

JeremiGendron commented:
Rebased @mvega3

Commit:
chore(psdk): update ark specs

https://github.com/ArkEcosystem/platform-sdk/blob/9ab9134da0e2221aa147daf67b61aee98f29cadf/packages/platform-sdk-ark/src/networks/ark/mainnet.ts

Commit:
chore(psdk): update ada specs

https://github.com/ArkEcosystem/platform-sdk/blob/058436ec500f17e5cd8a2909658fe94882de1295/packages/platform-sdk-ada/src/networks/ada/mainnet.ts

Commit:
chore(psdk): remove leftover xmr spec

Commit:
chore(psdk): add avax spec

Commit:
chore(psdk): add dot spec
